lib/vector_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative "quantum_exception"
+require_relative "extensions/extensions"
+
+# Contains help methods for classes which use operations above vectors
+# such as Qubit class
+module VectorHelper
+  include QuantumException
+
+  ACCURACY = 10
+  # Check if current vector is normalized: |x1| + |x2| + ... |xn| = 1
```

Comment:
there is a mistake:
# Check if current vector is normalized: |x1|^2 + |x2|^2 + ... |xn|^2 = 1

Reply:
fixed in 5ff6e41